lib/puppetlabs_spec_helper/module_spec_helper.rb

```diff
@@ -32,7 +32,7 @@ def verify_contents(subject, title, expected_lines)
       Puppet.settings[:stringify_facts] = false if ENV['STRINGIFY_FACTS'] == 'no'
       Puppet.settings[:trusted_node_data] = true if ENV['TRUSTED_NODE_DATA'] == 'yes'
     end
-    Puppet.settings[:strict_variables] = (ENV['STRICT_VARIABLES'] == 'yes' or (Puppet.version.to_f >= '4.0' and ENV['STRICT_VARIABLES'] != 'no'))
+    Puppet.settings[:strict_variables] = (ENV['STRICT_VARIABLES'] == 'yes' or (Puppet.version.to_f >= 4.0 and ENV['STRICT_VARIABLES'] != 'no'))
```

Comment:
combination of = and == makes it hard to read. But should be correct. I would prefer a different notation here.